pixi.toml

```diff
@@ -9,13 +9,20 @@ platforms = ["linux-64", "win-64", "osx-arm64", "osx-64"]
 [tasks]
 dev = "streamlit run --server.runOnSave=true app.py"
 deploy = "streamlit run --server.headless=true --global.developmentMode=false app.py"
+schema = "python app_config.py"
 
 [dependencies]
-python = "3.10"
+python = "3.12.*"
 pip = "23.3.2.*"
+pixi-pycharm = ">=0.0.6,<0.0.7"
```

Comment:
I don't think this is a run time dependency. Can you move it to `dev` or similar?

Reply:
yup 909e0ab